### PR TITLE
Replace links to w3c copy of canvas spec with links to living WHATWG spec

### LINKED ISSUE #267: Compositing needs to link to latest Canvas spec
As mentioned by @AmeliaBR  https://github.com/w3c/fxtf-drafts/issues/252#issuecomment-361461331

> (Aside: the links in the Compositing & Blending spec should probably be changed to point to the WHATWG Living Standard instead of the no-longer-updated W3C Canvas2D spec.)

### PR DESCRIPTION
**Pull request by chrishtr**
This fixes #267

### CODE REVIEW

**Comment by chrishtr** [comment]
Tab, the bibliography for the rendered spec still has a [2DCONTEXT2] in it that refers to w3.org's canvas spec. I don't know where that came from. Does bikeshed have a reference to it somehow?

File: compositing-1/Overview.bs [hunk 1]
```diff
@@ -14,7 +14,7 @@ Editor: Rik Cabanier, Adobe Systems Inc., cabanier@adobe.com, w3cid 106988
...
 Abstract: <ul><li>additional Porter Duff compositing operators</li><li>advanced blending modes which allow control of how colors mix in the areas where shapes overlap</li><li>compositing groups</li></ul>
-Abstract: In addition, this specification will define CSS properties for blending and group isolation and the properties of the <a href="https://www.w3.org/TR/2dcontext2/#dom-context-2d-globalcompositeoperation" title="canvas 2d globalcompositeoperation">‘globalcompositeoperation’</a> attribute as defined in [[2DCONTEXT2]].
+Abstract: In addition, this specification will define CSS properties for blending and group isolation and the properties of the <a href="https://html.spec.whatwg.org/multipage/canvas.html#dom-context-2d-globalcompositeoperation" title="canvas 2d globalcompositeoperation">‘globalcompositeoperation’</a> attribute as defined in [[2DCONTEXT2]].
```
Comment (tabatkins) [inline compositing-1/Overview.bs]: Replace this with `<<canvas/globalCompositeOperation>>`, and delete the biblio reference to the old W3C spec.

File: compositing-1/Overview.bs [hunk 2]
```diff
@@ -49,7 +49,7 @@ The 'background-blend-mode' property also builds upon the properties defined in
 
 This specification also enhances the rules as specified in <a href="https://www.w3.org/TR/2003/REC-SVG11-20030114/masking.html#SimpleAlphaBlending" title="simple alpha blending">Section 14.2 Simple alpha compositing</a> of [[SVG11]] and <a href="https://www.w3.org/TR/css3-color/#alpha" title="simple alpha compositing">simple alpha compositing</a> of [[!CSS3COLOR]].
 
-This module also extends the <a href="https://www.w3.org/TR/2dcontext2/#dom-context-2d-globalcompositeoperation" title="canvas 2d globalcompositeoperation">‘globalcompositeoperation’</a> as defined in [[!2DCONTEXT2]].
+This module also extends the <a href="https://html.spec.whatwg.org/multipage/canvas.html#dom-context-2d-globalcompositeoperation" title="canvas 2d globalcompositeoperation">‘globalcompositeoperation’</a> as defined in [[!2DCONTEXT2]].
```
Comment (tabatkins) [inline compositing-1/Overview.bs]: Replace with `<<canvas/globalCompositeOperation>>` here too.

File: compositing-1/Overview.bs [hunk 3]
```diff
@@ -324,7 +324,7 @@ Note that the gradient is not blending with the color of <a element>body</a>. In
 
 <h2 id="canvascompositingandblending">Specifying Compositing and Blending in Canvas 2D</h2>
 
-The <a href="https://www.w3.org/TR/2dcontext">canvas 2d</a> context defines the <a href="https://www.w3.org/TR/2dcontext/#dom-context-2d-globalcompositeoperation">‘globalCompositeOperation’</a> attribute that is used to set the current compositing and blending operator.
+The <a href="https://html.spec.whatwg.org/multipage/canvas.html#2dcontext">canvas 2d</a> context defines the <a href="https://html.spec.whatwg.org/multipage/canvas.html#dom-context-2d-globalcompositeoperation">‘globalCompositeOperation’</a> attribute that is used to set the current compositing and blending operator.
```
Comment (tabatkins) [inline compositing-1/Overview.bs]: Replace with `<<canvas/globalCompositeOperation>>` here too.

**Comment by tabatkins** [comment]
The section of the spec right after this diff, that defines the globalCompositeOperation attribute, is *totally* whackadoodle, too. It needs to be torn down and rewritten from scratch, as it's currently attempting to define a *CSS property* named globalCompositeOperation, with some random unrelated ID.  It also implies that the attribute should be parsed with the CSS parser (as it's defined with a CSS grammar), which I'm not sure is what we want.

File: compositing-1/Overview.bs (hunk 1 above)
Comment (chrishtr) [reply compositing-1/Overview.bs]: Done

File: compositing-1/Overview.bs (hunk 2 above)
Comment (chrishtr) [reply compositing-1/Overview.bs]: Done

File: compositing-1/Overview.bs (hunk 3 above)
Comment (chrishtr) [reply compositing-1/Overview.bs]: Done

**Comment by chrishtr** [comment]
Also updated compositing-2.

**Pull request merged**